tests/perf-test/GooglePlacesUtils.perf-test.js

```diff
@@ -0,0 +1,153 @@
+    s8ublocality: 'long_name',
+    a8dministrative_area_level_1: 'short_name',
+    p8ostal_code: 'long_name',
+    '8doesnt-exist': 'long_name',
+    s9ublocality: 'long_name',
+    a9dministrative_area_level_1: 'short_name',
+    p9ostal_code: 'long_name',
+    '9doesnt-exist': 'long_name',
+    s10ublocality: 'long_name',
+    a10dministrative_area_level_1: 'short_name',
+    p10ostal_code: 'long_name',
+    '10doesnt-exist': 'long_name',
+    s11ublocality: 'long_name',
+    a11dministrative_area_level_1: 'short_name',
+    p11ostal_code: 'long_name',
+    '11doesnt-exist': 'long_name',
+    s12ublocality: 'long_name',
+    a12dministrative_area_level_1: 'short_name',
+    p12ostal_code: 'long_name',
+    '12doesnt-exist': 'long_name',
+    s13ublocality: 'long_name',
+    a13dministrative_area_level_1: 'short_name',
+    p13ostal_code: 'long_name',
+    '13doesnt-exist': 'long_name',
+    s14ublocality: 'long_name',
+    a14dministrative_area_level_1: 'short_name',
+    p14ostal_code: 'long_name',
+    '14doesnt-exist': 'long_name',
+    s15ublocality: 'long_name',
+    a15dministrative_area_level_1: 'short_name',
+    p15ostal_code: 'long_name',
+    '15doesnt-exist': 'long_name',
+    s16ublocality: 'long_name',
+    a16dministrative_area_level_1: 'short_name',
+    p16ostal_code: 'long_name',
+    '16doesnt-exist': 'long_name',
+    s17ublocality: 'long_name',
+    a17dministrative_area_level_1: 'short_name',
+    p17ostal_code: 'long_name',
+    '17doesnt-exist': 'long_name',
+    s18ublocality: 'long_name',
+    a18dministrative_area_level_1: 'short_name',
+    p18ostal_code: 'long_name',
+    '18doesnt-exist': 'long_name',
+    s19ublocality: 'long_name',
+    a19dministrative_area_level_1: 'short_name',
+    p19ostal_code: 'long_name',
+    '19doesnt-exist': 'long_name',
+    s20ublocality: 'long_name',
+    a20dministrative_area_level_1: 'short_name',
+    p20ostal_code: 'long_name',
+    '20doesnt-exist': 'long_name',
+    s21ublocality: 'long_name',
+    a21dministrative_area_level_1: 'short_name',
+    p21ostal_code: 'long_name',
+    '21doesnt-exist': 'long_name',
+    s22ublocality: 'long_name',
+    a22dministrative_area_level_1: 'short_name',
+    p22ostal_code: 'long_name',
+    '22doesnt-exist': 'long_name',
+    s23ublocality: 'long_name',
+    a23dministrative_area_level_1: 'short_name',
+    p23ostal_code: 'long_name',
+    '23doesnt-exist': 'long_name',
+    s24ublocality: 'long_name',
+    a24dministrative_area_level_1: 'short_name',
+    p24ostal_code: 'long_name',
+    '24doesnt-exist': 'long_name',
+    s25ublocality: 'long_name',
+    a25dministrative_area_level_1: 'short_name',
+    p25ostal_code: 'long_name',
+    '25doesnt-exist': 'long_name',
+};
+
+test('getAddressComponents on a big dataset', async () => {
+    await measureFunction(
+        () => {
+            GooglePlacesUtils.getAddressComponents(addressComponents, bigObjectToFind);
+        },
+        {runs: 20},
+    );
+});
```

Comment:
How does this work? ie when does this fail? Would be nice to give some comment here to explain so people can quickly gather without goign to reassure documentation 😄

Reply:
you're right, let me update the PR with a little background and a link back to the docs!